### PR TITLE
Add calicoctl and crictl tenant clusters

### PR DESCRIPTION
**Pull request by ferrandinand**
* Add calicoctl and crictl and aliases etcdctl
* Moved to release v_4_1_0
* Changed CHANGELOG

### CODE REVIEW

**Comment by corest** [comment]
Can you also add etcdctl alias as in original CP ignition?

**Comment by ferrandinand** [comment]
> Can you also add etcdctl alias as in original CP ignition?

Should be solved now.

**Comment by calvix** [comment]
Did you test this? I think etcd LB uses port 2379

**Comment by calvix** [comment]
It might even differ across platforms so we should sue some kind of variable that might be already used in k8scloudconfig

**Comment by ferrandinand** [comment]
> Did you test this? I think etcd LB uses port 2379
Yes, it worked in gorgoth tenant master.

I saw in a previous PR you are removing LB in terraform, maybe we can focus on using same config so I will use local installation with 2379 port as it is done in other resources.

I need to rebase changes to put it in other vendor version, I will include this change also

File: v_4_0_0/master_template.go [hunk 1]
```diff
@@ -306,6 +306,18 @@ systemd:
       ExecStartPost=/usr/bin/systemctl restart k8s-kubelet.service
       [Install]
       WantedBy=multi-user.target
+  
+  - name: debug-tools.service
+    enabled: true
+    contents: |
+      [Unit]
+      Description=Calicoctl and crictl tools
```
Comment (kopiczko) [inline v_4_0_0/master_template.go]: Should it be following?

```suggestion
      Description=Install calicoctl and crictl tools
```

**Comment by kopiczko** [comment]
Also would be good to add an entry to `CHANGELOG.md`.

**Comment by calvix** [comment]
> Yes, it worked in gorgoth tenant master.

KVM and cloud tenants can be different, the port may be different, that's why its better to use variables that are filled by operators rather than hard-coding ports.

**Pull request merged**